LDAP-primer.md

```diff
@@ -308,22 +327,26 @@ changed to account for both:
     (|(uid=john.doe)(mail=john.doe))
 
 The filter would match an entry which has either `uid` or `mail` with the value "john.doe".
-In this case, the `uid` term would match. If the user entered "john.doe@example.com", the
+In this case, only the `uid` term would match. If the user entered "john.doe@example.com", the
 resulting filter would be:
 
     (|(uid=john.doe@example.com)(mail=john.doe@example.com))
 
-This filter would match the same entry, but in this case the `mail` term would evaluate
+This filter would match the same entry, but in this case only the `mail` term would evaluate
 to true.
 
+The above examples all use __equality__ filters. LDAP also supports other kinds of filters, notably
+__present__, __substrings__, __greaterOrEqual__, __lessOrEqual__, __approximate__, and
+__extensible__. These can all be combined with __or__ (as above), __and__ or __not__.
+
```

Comment:
Overload again. (I don't think that I've ever used an approximate match in anger.) I'll keep the paragraph but simplify it slightly.